protocol/network/7-properties.md

```diff
@@ -1,16 +1,105 @@
+7 Properties
+============
 
-# IPFS Protocol on the Wire
+## 7.1 Communication Model - Streams
 
-Authors: [Juan Benet](http://github.com/jbenet)
+The Network layer handles all the problems of connecting to a peer, and exposes
+simple bidirectional streams. Users can both open a new stream
+(`NewStream()`) and register a stream handler (`SetStreamHandler`). The user
+is then free to implement whatever wire messaging protocol she desires. This
+makes it easy to build peer-to-peer protocols, as the complexities of
+connectivity, multi-transport support, flow control, and so on, are handled.
```

Comment:
@diasdavid note that the above interface doesn't need to be everywhere, but the "protocol object" (returned by `Bind` or `Mount`) is a convenience wrapper around the `Node`'s methods, which is idiomatic Go.